src/objc/abstract/MCOAbstractMessage.mm

```diff
@@ -17,6 +17,10 @@
 #import "NSObject+MCO.h"
 #import "NSString+MCO.h"
 
+@interface MCOAbstractMessage ()
+- (id)init NS_DESIGNATED_INITIALIZER;
```

Comment:
Yeah... It has to be a part of the main interface or part of an anonymous class extension (here) for it to count.  I didn't want to put it in the main interface because it sends the wrong message, but I can move it back if you want.